app/resources/api/v2/custom_metadatum_collection_resource.rb

```diff
@@ -17,7 +17,11 @@ class CustomMetadatumCollectionResource < BaseResource
 
       # Attributes
       attribute :uuid, readonly: true
-      attribute :metadata, readonly: true
+      attribute :metadata #, readonly: true
```

Comment:
tiny thing, but I think you could delete rather than comment out - no reason to keep it I think